exercises/concept/making-the-grade/.docs/hints.md

```diff
@@ -5,11 +5,12 @@
 - `while` loops are used for _indefinite_ (uncounted) iteration
 - `for` loops are used for _definite_, (counted) iteration.
 - The keywords `break` and `continue` help customize loop behavior.
+- `std::to_string()` from `string` library can be used to convert a integer to string
```

Comment:
I would change integers to numbers. You already stated above, that doubles are possible as well.

Reply:
Thank you for the suggestion, it's done !